<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrmAccountSetup</class>
 <widget class="QMainWindow" name="FrmAccountSetup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>440</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>440</width>
    <height>440</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>440</width>
    <height>440</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_Cancel">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>400</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(225, 225, 225);</string>
    </property>
    <property name="text">
     <string>Thoát</string>
    </property>
    <property name="shortcut">
     <string>Esc</string>
    </property>
   </widget>
   <widget class="QToolBox" name="toolBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>421</width>
      <height>381</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>421</width>
       <height>221</height>
      </rect>
     </property>
     <attribute name="label">
      <string>Thông tin tài khoản đang sử dụng</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_AccountInfo">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>421</width>
        <height>231</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="title">
       <string/>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <widget class="QLabel" name="label_Info_Type">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>60</y>
         <width>259</width>
         <height>29</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(25, 149, 173);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_Info_Account">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>20</y>
         <width>259</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(25, 149, 173);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>47</x>
         <y>61</y>
         <width>91</width>
         <height>29</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Kiểu tài khoản:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>52</x>
         <y>20</y>
         <width>91</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Tên tài khoản:</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>421</width>
       <height>221</height>
      </rect>
     </property>
     <attribute name="label">
      <string>Tạo tài khoản mới</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>421</width>
        <height>231</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLineEdit" name="lineEdit_NewRePassword">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>140</y>
         <width>259</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::PasswordEchoOnEdit</enum>
       </property>
       <property name="placeholderText">
        <string>Viết liền, không dấu</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_NewType">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>60</y>
         <width>259</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="editable">
        <bool>false</bool>
       </property>
       <property name="currentText">
        <string>Student (Học sinh)</string>
       </property>
       <item>
        <property name="text">
         <string>Student (Học sinh)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Teacher (Giáo viên)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Root (Quản trị)</string>
        </property>
       </item>
      </widget>
      <widget class="QLineEdit" name="lineEdit_NewPassword">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>100</y>
         <width>259</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::PasswordEchoOnEdit</enum>
       </property>
       <property name="placeholderText">
        <string>Viết liền, không dấu</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_NewAccount">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>20</y>
         <width>259</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="placeholderText">
        <string>Viết liền, không dấu</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_ChangePassword_NewCreat">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>190</y>
         <width>259</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Tạo tài khoản</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>23</x>
         <y>138</y>
         <width>121</width>
         <height>35</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Nhập lại mật khẩu:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>75</x>
         <y>98</y>
         <width>71</width>
         <height>34</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Mật khẩu:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>48</x>
         <y>58</y>
         <width>91</width>
         <height>35</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Loại tài khoản:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>52</x>
         <y>18</y>
         <width>91</width>
         <height>35</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Tên tài khoản:</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>421</width>
       <height>221</height>
      </rect>
     </property>
     <attribute name="label">
      <string>Thay đổi mật khẩu</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>421</width>
        <height>231</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLineEdit" name="lineEdit_ChangePassword">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>20</y>
         <width>259</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::PasswordEchoOnEdit</enum>
       </property>
       <property name="placeholderText">
        <string>Viết liền, không dấu</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_ChangeNewPassword">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>60</y>
         <width>259</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="placeholderText">
        <string>Viết liền, không dấu</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_ChangeRePassword">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>100</y>
         <width>259</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="placeholderText">
        <string>Viết liền, không dấu</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>57</x>
         <y>19</y>
         <width>81</width>
         <height>33</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Mật khẩu cũ:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>49</x>
         <y>59</y>
         <width>91</width>
         <height>33</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Mật khẩu mới:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>23</x>
         <y>99</y>
         <width>121</width>
         <height>33</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Nhập lại mật khẩu:</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_ChangePassword">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>150</y>
         <width>259</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Đổi mật khẩu</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>421</width>
       <height>221</height>
      </rect>
     </property>
     <attribute name="label">
      <string>Xóa tài khoản</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>421</width>
        <height>221</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QPushButton" name="pushButton_Remove">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>150</y>
         <width>261</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Xóa tài khoản</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>42</x>
         <y>19</y>
         <width>101</width>
         <height>33</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Chọn tài khoản:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>23</x>
         <y>99</y>
         <width>121</width>
         <height>33</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Nhập lại mật khẩu:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>75</x>
         <y>59</y>
         <width>71</width>
         <height>33</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Mật khẩu:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_RemoveRePassword">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>100</y>
         <width>259</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::PasswordEchoOnEdit</enum>
       </property>
       <property name="placeholderText">
        <string>Viết liền, không dấu</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_RemoveAccount">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>20</y>
         <width>259</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="editable">
        <bool>false</bool>
       </property>
       <property name="currentText">
        <string/>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_RemovePassword">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>60</y>
         <width>259</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::PasswordEchoOnEdit</enum>
       </property>
       <property name="placeholderText">
        <string>Viết liền, không dấu</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>421</width>
       <height>221</height>
      </rect>
     </property>
     <attribute name="label">
      <string>Thiết lập mặc định</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>421</width>
        <height>201</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QPushButton" name="pushButton_RemoveDefault">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>150</y>
         <width>261</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Xóa về mặc định ban đầu</string>
       </property>
      </widget>
      <widget class="QPlainTextEdit" name="plainTextEdit">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>401</width>
         <height>121</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
         <italic>true</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(240, 240, 240);
color: rgb(0, 85, 127);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="plainText">
        <string>Chỉ có tài khoản với tên &quot;root&quot; có thể xóa được dữ liệu về mặc định của nhà sản xuất.
Sau khi thực hiện, toàn bộ tài khoản hiện tại sẽ được xóa khỏi cơ sở dữ liệu và các tài khoản mặc định sẽ được khôi phục.
Nếu không nhớ mật khẩu &quot;root&quot; hãy liên hệ với nhà sản xuất.</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit_NewAccount</tabstop>
  <tabstop>comboBox_NewType</tabstop>
  <tabstop>lineEdit_NewPassword</tabstop>
  <tabstop>lineEdit_NewRePassword</tabstop>
  <tabstop>pushButton_ChangePassword_NewCreat</tabstop>
  <tabstop>lineEdit_ChangePassword</tabstop>
  <tabstop>lineEdit_ChangeNewPassword</tabstop>
  <tabstop>lineEdit_ChangeRePassword</tabstop>
  <tabstop>pushButton_ChangePassword</tabstop>
  <tabstop>comboBox_RemoveAccount</tabstop>
  <tabstop>lineEdit_RemovePassword</tabstop>
  <tabstop>lineEdit_RemoveRePassword</tabstop>
  <tabstop>pushButton_Remove</tabstop>
  <tabstop>pushButton_Cancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
